Code/Framework/AzCore/Tests/Math/SimdMathTests.cpp

```diff
@@ -14,6 +14,76 @@ using namespace AZ;
 
 namespace UnitTest
 {
+    template<typename VectorType>
+    void TestToVec1()
```

Comment:
There are To/From Vec3 too later in the file, but since they are only under Vec4 it's not necessary to create a templated function for it. 

There are not To/From Vec4 functions.

Reply:
cool, thanks for the explanation